Server/VikingVault.Services/UserService.cs

```diff
@@ -105,6 +105,9 @@ public void DeleteUser(UserEmail userEmail)
             try
             {
                 User userToDelete = _dbContext.User.SingleOrDefault(user => user.Email == userEmail.Email);
+                _dbContext.Transactions.Where(transaction => transaction.Sender.Id == userToDelete.Id).Load();
+                _dbContext.Transactions.Where(transaction => transaction.Receiver.Id == userToDelete.Id).Load();
+
                 if(userToDelete == null)
```

Comment:
You should check for null before you get the transactions, because if it's null there is no point to check or his transactions